RELEASING.md

```diff
@@ -0,0 +1,99 @@
+# Releasing Sourcery
+
+There're no hard rules about when to release Sourcery. Release bug fixes frequently, features not so frequently and breaking API changes rarely.
+
+### Release
+
+Example is for releasing `0.6.1` version of the Sourcery. Make sure you've been added as owner for CocoaPods and have set up the api token to be able to upload releases on GitHub.
+
+Install dependencies.
+```
+bundle install
+bundle exec pod install
+```
+
+Run tests, check that all tests succeed locally and prepare the build.
+```
+rake tests
```

Comment:
The idea was to be sure that all tests are green before updating the metadata of the release. The same with the docs, basically to avoid any possible errors in `rake release:new`, but if you think it's not that bad, I don't mind to use only `rake release:new` task.